key.js

```diff
@@ -0,0 +1,45 @@
+/* global document */
+
+class Key {
+  constructor({
+    text = { en: 'a', ru: 'а' },
+    width = 1,
+    upperText = { en: 'A', ru: 'А' },
+    lastInRow = false,
+    code = 'KeyA',
+    extra = false
+  }, lang = 'en') {
+    this.text = text;
+    this.width = width;
+    this.upperText = upperText;
+    this.lastInRow = lastInRow;
+    this.code = code;
+    this.extra = extra;
+    this.lang = lang;
+  }
+
+  generate() {
+    const key = document.createElement('button');
+    key.classList.add('keyboard__key');
+
+    if (this.width === 2) {
+      key.classList.add('keyboard__key--wide');
+    } else if (this.width === 3) {
+      key.classList.add('keyboard__key--space');
+    }
+    if (this.extra === true) {
```

Comment:
No. The constructor sets its value based on `KEYBOARD_KEYS` array: if property `extra` defined as true `this.extra` is also true, otherwise it is false by default.

Reply:
so it's not necessary to compare with true, it can be only `if (this.extra)`